csaf_2.1/prose/edit/etc/section-label-to-display.json

```
@@ -242,7 +244,7 @@
   "requirement-23-mirror": "7.1.23",
   "requirement-3-tls": "7.1.3",
   "requirement-4-tlp-white": "7.1.4",
```

same as above comment. Tlp-clear?

```suggestion
  "requirement-4-tlp-clear": "7.1.4",
```